tests/warp/warp_test.go

```diff
@@ -102,6 +105,22 @@ var _ = ginkgo.BeforeSuite(func() {
 		},
 	)
 	gomega.Expect(err).Should(gomega.BeNil())
+
+	// Issue transactions to activate the proposerVM fork on the receiving chain
```

Comment:
should changes to this test be a separate PR?

Reply:
https://github.com/ava-labs/subnet-evm/pull/804